es/criteria/code-in-the-open.md

```diff
@@ -2,55 +2,55 @@
 
 ## Qué no hace
 
 * Hace el código fuente o política reusable.
 * Hace entendibles al mayor número de personas posible la codebase y el código que contiene.
 
-## Cómo testear
+## Cómo probar
 
-El origen de cada versión en uso es publicado en internet donde puede ser visto:
+La fuente de cada versión en uso es publicado en internet donde puede ser visto:
 
-* desde fuera de la organización que ha contribuido
-* sin la necesidad de forma alguna de autentificación o autorización
+* Desde fuera de la organización que ha contribuido.
+* Sin la necesidad de forma alguna de autentificación o autorización.
 
 Por cada commit, los revisores o reviewers verifican que el contenido no incluye informaicón sensible como pueden ser configuraciones, nombres de usuarios o contraseñas, claves públicas o cualquier otra credencial utilizada en sistemas en producción.
 
 ## Responsables políticos o legistadores: qué necesitan hacer
 
-* Desarrollar políticas en abierto.
-* Priorizar políticas en abierto y transparentes.
+* Desarrollar políticas de forma abierta.
+* Priorizar políticas abiertas y transparentes.
 
 ## Profesionales de la dirección de equipos: qué necesitan hacer
 
-* Desarrollar una cultura que abrace la apertura, el aprendizaje y la retroalimentación.
-* Colaborar con empresas externas y profesionales autónomos a través del trabajo en abierto.
+* Desarrollar una cultura que acoja la apertura, el aprendizaje y la retroalimentación.
+* Colaborar con empresas externas y profesionales autónomos a través del trabajo abierto.
 
 ## Profesionales del desarrollo de software y diseño: qué necesitan hacer
 
 * Dividir de forma clara los datos y el código, de tal manera que se pueda alcanzar el requerimiento sobre información sensible mencionado con anterioridad.
 
 ## Más información (en inglés)
 
-* [Mantener el código en abierto](https://gds.blog.gov.uk/2012/10/12/coding-in-the-open/) por el Servicio del Gobierno Digital del Reino Unido.
-* [Cuándo el código debería de estar en abierto y cuándo cerrado](https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed) by the UK Government Digital Service.
-* [Consideraciones de seguridad cuando se considere el código en abierto](https://www.gov.uk/government/publications/open-source-guidance/security-considerations-when-coding-in-the-open) por el Servicio del Gobierno Digital del Reino Unido.
+* [Desarrollnado código abierto](https://gds.blog.gov.uk/2012/10/12/coding-in-the-open/) por el Servicio del Gobierno Digital del Reino Unido.
+* [Cuándo el código debería ser abierto y cuándo cerrado](https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed) by the UK Government Digital Service.
+* [Consideraciones de seguridad cuando se considere el código abierto](https://www.gov.uk/government/publications/open-source-guidance/security-considerations-when-coding-in-the-open) por el Servicio del Gobierno Digital del Reino Unido.
```

Comment:
Why don't we avoid the verb consider twice and say instead "Consideraciones de seguridad cuando se desarrolla con código abierto"?